ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -925,6 +991,23 @@ def array_values(
     Returns
     -------
         A strategy that draws a list.
+
+    Examples
+    --------
+    >>> array_values(
+    ...     dtype=get_dtypes("float"),
+    ...     shape=st.shared(get_shape()),
```

Comment:
Let's eliminate the use of `st.shared` here not to cause confusion, also let's change the `get_dtypes` to have `valid` instead of `float`.